sfcasts/ep1/tdd.md

```diff
@@ -65,15 +78,15 @@ more time to make sure they're still passing:
 
 And... yes! We're still good to go!
 
-Before we move onto the *last* step in TDD, I think we need to add a couple more
```

Comment:
This is a long chapter. Ideally we would split it into 2. This would probably be the most obvious spot. Finish with something like:

> Next, let's finish this method the TDD-way: by writing more *test* for the missing features *first*. Then we'll move onto the final - and most fun step of TDD: refactoring!